etc/base.yaml

```diff
@@ -203,14 +200,11 @@ radar_rest_sources_backend:
     password: password
     ssl:
       enabled: false
-  # Clients example
-  #restSourceClients:
-  #  - sourceType: FitBit
-  #    authorizationEndpoint: https://www.fitbit.com/oauth2/authorize
-  #    tokenEndpoint: https://api.fitbit.com/oauth2/token
-  #    clientId: clientId
-  #    clientSecret: clientSecret
-  #    scope: activity heartrate sleep profile
+  restSourceClients:
+    fitbit:
+      enable: false
+      clientId: clientId
```

Comment:
do the original charts have these values?

Reply:
Indeed: https://github.com/RADAR-base/radar-helm-charts/blob/5a1535f3e913e68066c1a2a191d0f6d1d5a13ba1/charts/radar-rest-sources-backend/values.yaml#L116